store/postgres/src/relational_queries.rs

```diff
@@ -2527,9 +2527,26 @@ impl<'a> FilterCollection<'a> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct ChildKeyDetails<'a> {
+    pub parent_table: &'a Table,
+    pub parent_column: &'a Column,
+    pub child_table: &'a Table,
+    pub child_column: &'a Column,
+    pub column: &'a Column,
```

Comment:
This seemed to be clear until I saw `column` - this would be good to comment too, and explain what the fields are for.

Reply:
1cc9623d5 is it ok?

Reply:
Yes, that's much clearer!